src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs

```diff
@@ -513,7 +513,7 @@ public T[] ToArray()
         {
             // Short path if the bag is empty
             if (_headList == null)
-                return new T[0];
+				return Array.Empty<T>();
```

Comment:
@stephentoub if that's considered important, should we have a test for `Assert.NotSame(new ConcurrentBag<string>().ToArray(), ConcurrentBag<string>().ToArray())`?